boofuzz/utils/crash_binning.py

```diff
@@ -218,11 +218,15 @@ def import_file(self, file_name):
         @return:            self
         """
 
-        fh  = open(file_name, "rb")
-        tmp = pickle.loads(zlib.decompress(fh.read()))
-        fh.close()
-
-        self.bins = tmp.bins
+        self.bins = {}
+        with open(file_name, "rb") as fh:
+            bin_dict = json.loads(fh.read())
```

Comment:
another elegant way:

    bin_dict = json.load(open(file_name, "rb"))

Reply:
Neat! As with the last one I'll leave it up to @aedrax whether to make this change since he's focusing on Py2/Py3. I was the one who wrote this code so my bad.